packit/utils/repo.py

```diff
@@ -485,3 +486,49 @@ def commit_message_file(
             # so a plain Git command can be used here.
             git.cmd.Git().interpret_trailers(*args, fp.name)
         yield fp.name
+
+
+def get_commit_message_from_action(
+    output: Optional[List[str]], default_title: str, default_description: str
+) -> Tuple[str, str]:
+    """
+    Parse the output of the commit action and in case the action is not defined,
+    no output has been produced or it couldn't be parsed, return the defaults.
+
+    Args:
+        output: Output produced by the `commit-message` action.
+        default_title: Commit title that is used in case the produced commit
+            message hasn't been produced, is malformed or couldn't be parsed.
+        default_description: Commit description that is used in case the
+            produced commit message hasn't been produced, is malformed or
+            couldn't be parsed.
+
+    Returns:
+        Pair of the commit title and commit message that will be used.
+    """
+    # no output has been produced, or action doesn't exist
+    if not output:
+        return (default_title, default_description)
+
+    # split by the divider
+    split_output = "\n".join(output).rsplit(sep=COMMIT_ACTION_DIVIDER, maxsplit=1)
```

Comment:
I do not like that much that we will require users outputting this divider event if they do not need any debugging output before, would it make sense allow not requiring this:
1. if not present, treat it directly as commit title + description
2. if present, do what is done currently

or do you think this would bring more confusion?

Reply:
I was considering it and left it exactly because of the confusion… though let me add some tests to see if it would require too many changes :thinking:

Reply:
Looks OK, I'll update the docs, keep it in a separate commit for a reference